include/sys/cdefs.h

```diff
@@ -182,4 +182,21 @@
  */
 #define __arraycount(__x) (sizeof(__x) / sizeof(__x[0]))
 
+/* __BIT(n): nth bit, where __BIT(0) == 0x1. */
+#define __BIT(__n)                                                             \
+  (((uintmax_t)(__n) >= NBBY * sizeof(uintmax_t))                              \
+     ? 0                                                                       \
+     : ((uintmax_t)1 << (uintmax_t)((__n) & (NBBY * sizeof(uintmax_t) - 1))))
```

Comment:
The code is taken from NetBSD.

I guess they use '&' simply because shifting by more bits than there're in the word gives undefined behaviour according to C standard.

Reply:
I'd leave it as is.